client/api_handler.py

```diff
@@ -0,0 +1,19 @@
+from constants import UrlTypes
+from scraper import general_metadata, wikipedia_metadata, youtube_metadata
+import url_utils
+
+
+def get_metadata(url):
+    response = url_utils.get_url_data(url)
+    scraper = get_scraper(response)
+    return scraper.parse_content(response)
+
+
+def get_scraper(response):
```

Comment:
reference to other comment in parse_content, instead of getting scrapper, you're creating it.
I think this should be named
create_scrapper